fire/io/bernese/bernese.py

```diff
@@ -0,0 +1,314 @@
+                    yx=temp_table[entry]["YX"],
+                    zx=temp_table[entry]["ZX"],
+                    zy=temp_table[entry]["ZY"],
+                )
+
+    def addneq_parse(self, addneq_data: list) -> None:
+        """
+        Parse linjerne fra en ADDNEQ fil.
+        """
+        # Vi finder observationslængdeafsnittet ved at finde indekser ud fra kendt rækkefølge af faste headers
+        første_sektion_begyndelse = (
+            addneq_data.index(
+                " Sol Station name         Typ Correction  Estimated value  RMS error   A priori value Unit    From                To                  MJD           Num Abb  \n"
+            )
+            + 2
+        )
+        første_sektion_ende = addneq_data.index(
+            " Site-specific troposphere parameters:\n"
+        )
+        første_sektion = addneq_data[første_sektion_begyndelse:første_sektion_ende]
+
+        # Det samme gør sig gældende med afsnittet om spredninger
+        anden_sektion_begynd = (
+            addneq_data.index(" Comparison of individual solutions:\n") + 3
+        )
+        anden_sektion_ende = addneq_data.index(
+            " Variance-covariance scaling factors:\n"
+        )
+        anden_sektion = addneq_data[anden_sektion_begynd:anden_sektion_ende]
+
+        # Så skal der udtrækkes observationslængder fra den første udklippede sektion
+        for line in første_sektion:
+            if not line.isspace():  # skip evt. tomme linjer
+                observation = addneq_parse_obslengthline(line)
+                delta = datetime.strptime(
+                    observation["TO"], "%Y-%m-%d %H:%M:%S"
+                ) - datetime.strptime(observation["FROM"], "%Y-%m-%d %H:%M:%S")
+                self.resultatset.stations[observation["STATION NAME"]].obslength = delta
+
+        # Og spredning fra den anden udklippede sektion
+        stations_rms = {}
+        for station in self.resultatset.stations.keys():
+            stations_rms[station] = {}  # opret en tabel med navne fra CRD parsing
+        for line in anden_sektion:
+            if not line.isspace():  # skip evt. tomme linjer
+                spredningsset = addneq_parse_spreadline(line)
+                if spredningsset["DIRECTION"] == "N":
+                    stations_rms[spredningsset["STATION NAME"]]["N"] = spredningsset[
+                        "SPREAD"
+                    ]
+                elif spredningsset["DIRECTION"] == "E":
+                    stations_rms[spredningsset["STATION NAME"]]["E"] = spredningsset[
+                        "SPREAD"
+                    ]
+                elif spredningsset["DIRECTION"] == "U":
+                    stations_rms[spredningsset["STATION NAME"]]["U"] = spredningsset[
+                        "SPREAD"
+                    ]
+        for station in stations_rms:
+            if stations_rms[station] != {}:  # glem stationer uden værdier
+                n = stations_rms[station]["N"]
```

Comment:
Her kan du også spare et indentationsniveau:
```suggestion
        for station in stations_rms:
            if stations_rms[station] == {}:  # spring over stationer uden værdier
                continue
            n = stations_rms[station]["N"]
            ...
```

Reply:
Done.